moderation/commands.go

```diff
@@ -1396,6 +1401,12 @@ type MessagesWithAttachmentsFilter struct{}
 func (*MessagesWithAttachmentsFilter) Matches(msg *dstate.MessageState) (delete bool) {
 	return len(msg.GetMessageAttachments()) > 0
 }
+// Only delete messages with attachments.
+type BotMessagesOnly struct{}
```

Comment:
```suggestion

// Only delete bot messages.
type BotMessagesFilter struct{}
```
Fix the comment and make the struct name consistent with the other filters.

Reply:
Okay, changed. Thank you :3